### DISCUSSION TITLE
Can someone else verify that xschedule works in master?

### DISCUSSION THREAD

**Discussion by derwin12** [xLightsSequencer/xLights#4489] [Q&A]
When it tries to play a song in master (works in prior nightly) it just throws a codec error and closes.
So set the schedule to an earlier time, start xschdule, it says "NOW" for playing but then crashes out in vs2022

**Comment by keithsw1111** [comment]
It seems to play for me

**Comment by derwin12** [comment]
I am getting exceptions when it tries to play the audio. I will try the nightly when it comes thru.

**Comment by derwin12** [comment]
This line throws it..
`    avcodec_parameters_to_context(codecContext, audioStream->codecpar);`

![image](https://github.com/xLightsSequencer/xLights/assets/4643499/c9161daa-b198-48ad-ae8e-1ec697968388)

**Comment by derwin12** [comment]
looks like it wants avcodec-60.dll which I dont have. Not sure if I did that somehow, but let me see if I can get that.
